apps/condo/domains/meter/hooks/useMeterTableColumns.tsx

```diff
@@ -0,0 +1,172 @@
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core'
+import get from 'lodash/get'
+import { getDateRender, getTextRender } from '../../common/components/Table/Renders'
+import { useIntl } from 'react-intl'
+import React, { CSSProperties, useCallback, useState } from 'react'
+import { InputNumber } from 'antd'
+import { colors } from '@condo/domains/common/constants/style'
+import { fontSizes } from '@condo/domains/common/constants/style'
+import pickBy from 'lodash/pickBy'
+import { isEmpty } from 'lodash'
+
+const inputNumberCSS = css`
+  & .ant-input-number-handler-wrap {
+    visibility: hidden;
+  }
+
+  font-size: ${fontSizes.label};
+  
+  width: 100%;
+  padding-right: 20%;
+`
+
+const INPUT_CONTAINER_STYLE: CSSProperties = { position: 'relative' }
+const inputMeterReadingFormatter = value => value.toString().replace(',', '.')
+const inputMeterReadingParser = input => input.replace(/,+/g, '.')
```

Comment:
it doen't look like a parser. it's a formatter too :) and shouldn't we replace multiple ... too ?

also could you please move static regexp to a constant.

Reply:
1) These are functions that are passed as `parser` and` formatter` props to InputNumber. I think the names are appropriate here
2) ok
3) ok